tools/scripts/bumpenvs.yaml

```diff
@@ -142,6 +142,18 @@ tf2_gpu_hashed:
 tf2_gpu_versioned:
   new: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.5-tf-2.4-gpu-0.17.10
   old: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.3-tf-2.4-gpu-0.17.9
+deepspeed_gpu_hashed:
+  new: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.5-tf-2.4-gpu-5158dec
```

Comment:
`deepspeed_gpu_hashed` and `deepspeed_gpu_versioned` seem like duplicate images of other things... or am I misunderstanding?